Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrVisitor.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+package org.jetbrains.kotlin.native.interop.gen
+
+interface StubIrVisitorAux<T, R> {
```

Comment:
So now it can be renamed to `StubIrVisitor`.
And `aux` parameters could be more like `data` (like in backend IR visitors).